tests/main/interfaces-netlink-audit/task.yaml

```diff
@@ -0,0 +1,39 @@
+summary: Ensure that the netlink audit interface works.
+
+details: |
+    The netlink-audit interface allows read/write to kernel audit system.
+
+    The test-snapd-netlink-audit snap creates a netlink socket and binds it. 
+
+prepare: |
+    . $TESTSLIB/snaps.sh
+
+    # Install a snap declaring a plug on netlink-audit
+    install_local test-snapd-netlink-audit
+
+execute: |
+    CONNECTED_PATTERN=":netlink-audit +test-snapd-netlink-audit"
```

Comment:
Nitpick, why don't we put those into environment?

Reply:
done